holmes/utils/pydantic_utils.py

```
@@ -52,3 +58,90 @@ def load_model_from_file(
+            example_value = examples[0]
+
+        if example_value is None:
+            default = getattr(field_info, "default", PydanticUndefined)
+            default_factory = getattr(field_info, "default_factory", None)
+
+            if default is not PydanticUndefined:
+                example_value = default
+            elif default_factory is not None:
+                try:
+                    example_value = default_factory()
+                except Exception:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)